CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ PyO3 versions, please see the [migration guide](https://pyo3.rs/master/migration
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Add support for building for CPython limited API. [#1152](https://github.com/PyO3/pyo3/pull/1152)
+
+### Changed
```

Comment:
I would also like to see the point about requiring `#[pyclass(subclass)]` here, as I think it's potentially quite breaking to some user code:

```suggestion
### Changed
- `#[pyclass(subclass)]` is now required for subclassing from Rust (was previously just required for subclassing from Python). [#1152](https://github.com/PyO3/pyo3/pull/1152)
```